client/signup/steps/user/index.jsx

```diff
@@ -350,6 +369,12 @@ export class UserStep extends Component {
 		return this.props.step && 'completed' === this.props.step.status;
 	}
 
+	userCreationCompletedWithAndHasHistory( props ) {
+		return (
+			props.step && 'completed' === props.step.status && detectHistoryNavigation.loadedViaHistory()
```

Comment:
We could simplify this by using optional chaining:

```suggestion
			'completed' === props.step?.status && detectHistoryNavigation.loadedViaHistory()
```